src/molecules/Hero/index.js

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import Button from "../../atoms/Button"
+import ComputerImage from "../../images/computer-line@2x.png"
+import Photography from "../../atoms/Photography"
+import styles from "./hero.module.css"
+
+const Hero = () => (
+  <div>
+    <Photography></Photography>
+    <div className={styles.hero}>
+      <div className={styles.text}>
+        <h1>Aprenda programação gratuitamente.</h1>
```

Comment:
Extract inner text to a property, like `title`.

Reply:
Done.